src/index.js

```diff
@@ -23,6 +24,13 @@ export default function* () {
 
   } else {
     const path = yield resolve({ file, name: "Flyfile" })
+    const ext = path.split(".")[path.split(".").length - 1]
+    const modules = jsVariants["." + ext]
+    let extModule = modules
+    if(Array.isArray(modules)) {
+      extModule = modules[0]
+    }
+    require(extModule)
```

Comment:
I wonder how this would work for more complicated transformers like babel that use yet other dependencies to support features like _generators_, etc.